sdcm/tester.py

```diff
@@ -3089,6 +3084,18 @@ def tearDown(self):
         self._check_alive_routines_and_report_them()
         self._check_if_db_log_time_consistency_looks_good()
 
+    @pytest.fixture(autouse=True, name='setup_logging')
+    def fixture_setup_logging(self):
```

Comment:
why do we have test specific functions in Tester? Isnt Tester used for all of our normal SCT stuff as well?

Reply:
for some of the mechanize to work, they were needed to move into fixture, all of their dependencies need to be a fixture.

and we are using Tester in all kind of way in k8s functional tests, and in unit tests.

Reply:
Oh no, I just learned the extend of our usage of unittest in all of our SCT runs :-( Nvm then, this is fine here, but it gave me serious nightmares about design of our testcases